test/extensions/filters/network/thrift_proxy/translation_integration_test.cc

```diff
@@ -0,0 +1,195 @@
+#include "envoy/config/filter/network/thrift_proxy/v2alpha1/thrift_proxy.pb.h"
+
+#include "extensions/filters/network/well_known_names.h"
+
+#include "test/extensions/filters/network/thrift_proxy/integration.h"
+#include "test/test_common/network_utility.h"
+
+#include "absl/strings/ascii.h"
+#include "gtest/gtest.h"
+
+using testing::Combine;
+using testing::TestParamInfo;
+using testing::TestWithParam;
+using testing::Values;
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace ThriftProxy {
+namespace {
+std::string thrift_config;
```

Comment:
Why not make this a member of the test fixture below? The name should be `thrift_config_` in any case, to prevent confusion with local variables.